src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,160 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private int localSize;
+    private Object[] values;
+
+    public ArrayList() {
+        localSize = 0;
+        values = new Object[DEFAULT_SIZE];
+    }
+
+    public ArrayList(int newSize) {
+        localSize = newSize;
+        values = new Object[localSize];
+    }
```

Comment:
```suggestion
    public ArrayList(int initialCapacity) {
        values = new Object[initialCapacity];
    }
```
ти всеодно створюєш пустий ліст, просто з початковим розміром іншим, але в ньому нема жодних елементів